requirements/runtime.txt

```diff
@@ -7,6 +7,7 @@ lpips
 mmcv>=2.0.0rc1
 mmengine
 numpy
+open_clip_torch
```

Comment:
we have `CLIP` installed in `requirements/optional.txt`. Does it have the same function as `open_clip_torch`?

Reply:
We need both, two packages are different. Both moved to `optional.txt`